pytest_instafail.py

```diff
@@ -46,7 +46,8 @@ def pytest_collectreport(self, report):
         # Show errors occurred during the collection instantly.
         TerminalReporter.pytest_collectreport(self, report)
         if report.failed:
-            self.rewrite("")  # erase the "collecting" message
+            if self.isatty:
```

Comment:
It seems `isatty` was added to `TerminalReporter` in pytest 2.9. That's why tests are failing on older pytests. Can you add a `hasattr` check here, so that the rewrite is always done on older pytests.